otter/auth.py

```diff
@@ -284,6 +284,42 @@ def __hash__(self):
                      self._url, self._admin_url))
 
 
+@implementer(IAuthenticator)
+class SingleTenantAuthenticator(object):
+    """
+    An authentication handler that authenticates using a single tenant id.
+    """
+    def __init__(self, _identity_user, _identity_password, url):
```

Comment:
Since this class authenticates only one tenant I think it should take that tenant ID as argument here and ensure only that is the only tenant allowed in `authenticate_tenant`